app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+  before_action :authenticate_user!
+
+  def index
+    @rents = Rent.all
+    render_paginated @rents
+  end
+
+  def create
+    @rent = Rent.new(rent_params)
+    @rent.save
```

Comment:
What would happend if you pass the incorrect params to rent and the model cannot be created? How do you inform the user about the problem?